modules/serverless-install-web-console.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * /serverless/installing_serverless/installing-openshift-serverless.adoc
+
+[id="serverless-install-web-console_{context}"]
+= Installing the {ServerlessOperatorName}
+
+This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the OperatorHub using the {product-title} web console.
+
+.Procedure
+
+. Navigate in the web console to the *Catalog* → *OperatorHub* page.
+. Scroll, or type they keyword *Serverless* into the *Filter by keyword box* to find the {ServerlessOperatorName}.
++
+image::serverless-search.png[{ServerlessOperatorName} in the {product-title} web console]
```

Comment:
This image shows a tile with an "already installed" Serverless (see the tick). Not sure if this is a big problem but it's not exactly what the user will see.